epsagon/__init__.py

```diff
@@ -28,7 +28,7 @@ def dummy_wrapper(func):
     step_lambda_wrapper = dummy_wrapper  # pylint: disable=C0103
     azure_wrapper = dummy_wrapper  # pylint: disable=C0103
     python_wrapper = dummy_wrapper  # pylint: disable=C0103
-    EpsagonFlask = dummy_wrapper  # pylint: disable=C0103
```

Comment:
export with both name, maybe add a deprecation warning

Reply:
EpsagonFlask is not listed in the `__all__`, So I don't think that anyone is doing `from epsagon import EpsagonFlask`